### PR TITLE
Dynamic changes 3

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

read cl

~- add: Thieves' pacifism implant is now an actual thing that can be removed.~
~- add: Added pacifism implants in CMO locker.~ later...
:cl:
- tweak: Dynamic gamemode threat and balance now scales with the amount of players.
- add: Brought back traitor flavor character text.
- add: On polymorph ghosts will start properly following the person.
- tweak: Ling got better... in a way
- remove: Removed transform sting until polymorph gets good.
- tweak: Made midround antags rarer on secret.

### CODE REVIEW

**Comment by whateverusername0** [comment]
SUBMODULE UPDATE DETECTED :goberserk:

**Review by Piras314 — APPROVED** [review]

**Comment by Piras314** [comment]
needs submodule fixed, yaml linter etc etc

File: RobustToolbox [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 32bca7cfd417edcad9a60c2b1703eba8675f56af
+Subproject commit d1d43f834b8845da698ef1898e8191ab159ee367
```
[the remark this answers is not in the document]
Comment (whateverusername0) [reply RobustToolbox:1]: ok what do i do to make it disappear

**Comment by whateverusername0** [comment]
SUBMODULE UPDATE CHECKED :godmode:

**Comment by Piras314** [comment]
conflicts @whateverusername0

**Pull request merged**